tetris.py

```diff
@@ -297,35 +297,42 @@ def full_line_clear(self):
         """
         Clears the line once it's fully populated with blocks
         """
+        flash_list = []
         y_coordinates = [y for (x, y) in self.coord_extractor()]
         coordinates_counter = collections.Counter(y_coordinates)
         for x_line in range(game_height):
             count = coordinates_counter[x_line]
             if count == game_width:
-                for flash in range(2):
-                    self.flasher(x_line, 'yellow')
-                    self.canvas.update()
-                    time.sleep(0.1)
-                    self.flasher(x_line, 'brown')
-                    self.canvas.update()
-                    time.sleep(0.1)
-                self.flasher(x_line, D_GREY)
-                for letter, coord_list in self.landed_blocks.items():
-                    # self.landed_blocks = {letter: [(a, b) for (a, b) in coord_list if b > x_line] + [(a, b+1) for (a, b) in coord_list if b < x_line]} #Why this doesn't work?
-                    self.landed_blocks[letter] = [(a, b) for (a, b) in coord_list if b > x_line] + [
-                        (a, b + 1) for (a, b) in coord_list if b < x_line
+                flash_list.append(x_line)
+        if x_line in flash_list:
```

Comment:
This check is unnecessary and doesn't do what you think it does. Delete it.

Reply:
Changed to if flash_list: